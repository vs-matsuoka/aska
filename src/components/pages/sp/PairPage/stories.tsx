import { Story, Meta } from '@storybook/react';
import PairPage from '.';

export default {
  title: 'PairPage',
  component: PairPage
} as Meta;

export const Default: Story = () => (
  <PairPage
    index={1}
    kirinukiSrc="/SP/Pair_Detail/Kirinuki/s32_Entry_Kirinuki_01.png"
    nameSrc="/SP/Pair_Detail/Name/s32_Pair_name_01.png"
    baseASrc="/SP/Pair_Detail/HNBase/s32_Pair_HNBase_01_A.png"
    baseBSrc="/SP/Pair_Detail/HNBase/s32_Pair_HNBase_01_B.png"
    name="Senpai Club"
  />
);
