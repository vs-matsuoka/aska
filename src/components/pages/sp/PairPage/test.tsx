import { render, screen } from '@testing-library/react';

import PairPage from '.';

describe('<PairPage />', () => {
  it('should render the heading', () => {
    const { container } = render(
      <PairPage
        index={1}
        kirinukiSrc="/SP/Pair_Detail/Kirinuki/s32_Entry_Kirinuki_01.png"
        nameSrc="/SP/Pair_Detail/Name/s32_Pair_name_01.png"
        baseASrc="/SP/Pair_Detail/HNBase/s32_Pair_HNBase_01_A.png"
        baseBSrc="/SP/Pair_Detail/HNBase/s32_Pair_HNBase_01_B.png"
        name="Senpai Club"
      />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
