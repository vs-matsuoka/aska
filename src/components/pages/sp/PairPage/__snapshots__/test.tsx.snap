// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PairPage /> should render the heading 1`] = `
<div
  class="fixed emotion-0"
>
  <div
    class="h-screen w-screen"
  />
  <img
    alt="background"
    class="select-none"
    data-nimg="fill"
    decoding="async"
    loading="lazy"
    sizes="100vw"
    src="/_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=3840&q=75"
    srcset="/_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=640&q=75 640w, /_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=750&q=75 750w, /_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=828&q=75 828w, /_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=1080&q=75 1080w, /_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=1200&q=75 1200w, /_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=1920&q=75 1920w, /_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=2048&q=75 2048w, /_next/image?url=%2FSP%2FPair_Detail%2Fs32_Entry_BG.jpg&w=3840&q=75 3840w"
    style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
  />
</div>
`;
